tests/test_migrations.py

```diff
@@ -0,0 +1,181 @@
+import random
+
+import pytest
+
+
```

Comment:
See https://github.com/wemake-services/django-test-migrations for more on this neat plug-in, `django-test-migrations`!